build/common/commands/new.py

```diff
@@ -96,6 +96,11 @@ def main():
         command = mysql_command + [f"GRANT ALL PRIVILEGES ON `{db_name}`.* TO '{db_name}'@'%'; FLUSH PRIVILEGES;"]
```

Comment:
I think this command shouldn't run if `rds_db` is set since RDS does not allow running `GRANT ALL`

Reply:
so that'll fail, instead we could have something like:

```py
grant_privileges = RDS_PRIVILEGES if config.get(RDS_DB) else "ALL PRIVILEGES"
command = mysql_command + [f"GRANT {grant_privileges} ON `{db_name}`.* TO '{db_name}'@'%'; FLUSH PRIVILEGES;"]
```

and inside `constants.py`

```py
RDS_PRIVILEGES = "SELECT, INSERT, UPDATE, DELETE, CREATE, DROP, INDEX, ALTER, CREATE TEMPORARY TABLES, CREATE VIEW, EVENT, TRIGGER, SHOW VIEW, CREATE ROUTINE, ALTER ROUTINE, EXECUTE, LOCK TABLES"